.circleci/config.yml

```diff
@@ -9,28 +9,26 @@ executors:
 
 aliases:
   e2e-executor: &e2e-executor
-    executor:
-      name: node
-      image: 10-browsers
+    docker:
+      - image: cypress/base:10
 
-  restore_node_modules: &restore_node_modules
+  restore_cache: &restore_cache
     restore_cache:
       name: Restore node_modules cache
       keys:
-        - node-modules-{{ checksum "yarn.lock" }}
+        - yarn-cypress-cache-{{ checksum "yarn.lock" }}
 
   install_node_modules: &install_node_modules
     run:
       name: Install node modules
-      command: |
-        yarn
+      command: yarn
```

Comment:
@LekoArts thoughts on using `--frozen-lockfile` here? Think there's any harm?

Reply:
I think it doesn't hurt, right?
Arguments like [these](https://github.com/yarnpkg/website/issues/700#issuecomment-340993837) seem reasonable - but I didn't use it too long myself so don't know how it behaves at a larger scale.